web/src/components/Spoiler/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
C React 17 не нужен импорт
```suggestion
import { memo } from 'react';
```